cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -134,19 +134,35 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
       query("SHOW DATABASES", assertDatabasesShown) {
         resultTable()
       }
-      p("This command is optionally idempotent, with the default behavior to throw an exception if the database already exists. " +
-        "Appending `IF NOT EXISTS` to the command will ensure that no exception is thrown and nothing happens should the database already exist. " +
-        "Adding `OR REPLACE` to the command will result in any existing database being deleted and a new one created.")
-      query("CREATE DATABASE customers IF NOT EXISTS", ResultAssertions(r => {
-        assertStats(r, systemUpdates = 0)
-      })) {}
-      query("CREATE OR REPLACE DATABASE customers", ResultAssertions(r => {
-        assertStats(r, systemUpdates = 2)
-      })) {
-        p("This is equivalent to running `DROP DATABASE customers IF EXISTS` followed by `CREATE DATABASE customers`.")
+      section("Handling Existing Databases", "administration-databases-create-database-existing", "enterprise-edition") {
```

Comment:
Currently these new sections does not show up in any menus, not sure if we would like that?

Reply:
Added to the top of this file